content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
```

Comment:
hello @MikeSpreitzer . A few initial comments:
- It may be helpful to change the headings, `Outline` and `Status`, to something more meaningful.
This first section could use the heading, `Overview`.
- This feature is called `RequestManagement`? You could use this name to create the title, such as `API Server Request Management`, or something similar.
- There are other docs pages that describe how to enable a feature. Perhaps use similar language to describe enabling the feature. One example: https://kubernetes.io/docs/reference/using-api/api-concepts/#enable-the-server-side-apply-alpha-feature 
- It looks like the feature is already listed in the feature gates table. Check that the version and definition summary are correct? https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
- Could this page link to definitions for `WATCHes` and `EXECUTes`?

Reply:
Yes, this feature is called `RequestManagement` but that was a compromise name adopted in early discussions before we really understood this work well.  I think "API priority and fairness" is a better description; "request management" is very vague and could include the functionality of this feature and/or other functionality.  So I would not like the poor choice of feature name to influence the doc any more than necessary.  In fact, I might even suggest changing the feature name.

Reply:
`kubernetes EXEC` (in the API) sense seems a hard thing to find via web search. I don't think there's a page about it in the Kubernetes docs. If there's a page anywhere on the web I can try to read that page, understand the contnts, and add something about that to the Kubernetes docs.

(I don't know much about Kubernetes at that level of API detail).

Reply:
It's definitely a good idea to use a feature-state shortcode and to hyperlink to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Maybe in the future there'll be more automation so that the actual feature status from [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/) gets copied to the relevant parts of the website. Right now that's not in place.
The website has had, and still has, pages that describe the status of a feature the last time that a particular page was touched. At the time of writing [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#limitations) was a good example of this, talking about versions back to v1.5 (!)

With that in mind, I favor being a little DRY and leaving the status information for the feature gates page.